back/spec/services/permissions/project_permissions_service_spec.rb

```diff
@@ -632,9 +723,10 @@
 
       # First check project length sure all the 'projects' queries are preloaded
       expect(projects.length).to eq 5
+      user_requirements_service = Permissions::UserRequirementsService.new(check_groups: false)
```

Comment:
I don't understand how these tests passed before, without recycling the `UserRequirementService` here. There are 5 projects, and for each project, at least 3 queries need to be made in the `UserRequirementService`: `Topic.where(include_in_onboarding: true).count` (`count` always performs a query, so I replaced it by `size`, which is sometimes cached), `Area.where(include_in_onboarding: true).size` and `CustomField.registration.enabled.order(:ordering)`. Without recycling the UserRequirementService, I don't understand how you could have fewer than 5 x 3 = 15 queries.